docs/invoking-custom-code.md

```diff
@@ -33,7 +33,8 @@ A.CallTo(()=>fakeShop.NumberOfSweetsSoldOn(A<DateTime>._))
  .Invokes((DateTime when) => System.Console.Out.WriteLine("showing sweet sales for " + when))
  .Returns(17);
 
-// Pass an IFakeObjectCall into the creation method for more advanced scenarios.
+// Pass an IFakeObjectCall into the creation method for more advanced scenarios,
+// including configuring methods that have more than 4 parameters.

```

Comment:
Why "creation"? I think it was copied from throwing-exceptions.md. It's a callback method here.

Reply:
Sigh. I eyeballed that on the "throwing" docs and eventually became happy and didn't even notice here! Thanks. Will just amend.